persistence/db.go

```diff
@@ -66,14 +69,26 @@ func (pg *PostgresContext) ResetContext() error {
 }
 
 // TECHDEBT: Implement proper connection pooling
```

Comment:
Is it safe to remove this TECHDEBT line item?

Reply:
Hi @Olshansk, Thanks for reviewing. I was just going through the code and found this Techdebt item which was not resolved so worked on that. I will go through the review and will make changes accordingly, after that we can remove this line.

Reply:
💯